src/lib/flow/include/sol-flow-parser.h

```diff
@@ -107,16 +108,15 @@ int sol_flow_parser_del(
  * @brief Parses a flow description contained in @a buf and returns the resulting node type.
  *
  * @param parser The Flow Parser handle
- * @param buf Flow description to be parsed
- * @param len Buffer's size
+ * @param buf Flow description to be parsed. @note The buffer will be
```

Comment:
i did not noticed that. I'll remove the `@note` from there. I prefer this than a `@note` in the final because it's clear and the this info is related with the parameter.